lib/classes/Utils.ts

```diff
@@ -8,7 +8,7 @@ export class Utils {
      * @param {string} imageURL
      * @param {SearchParameters} params?
      */
-    handleOptionalParameters(baseURL: string, imageURL: string, params?: SearchParameters) {
+    handleOptionalParameters(baseURL: string, imageURL?: string, params?: SearchParameters) {
```

Comment:
what does this fix?

Reply:
compiling `strictNullChecks` set to `true`

Reply:
i see